code/game/objects/structures/airlock_assembly.dm

```diff
@@ -22,9 +22,10 @@
 	var/airlock_type = "generic" //the type path of the airlock once completed
 	var/glass = AIRLOCK_NOGLASS // see defines
 	var/created_name = null
+	var/width = 1 //Used for multitile assemblies
```

Comment:
```suggestion
	/// Used for multitile assemblies
	var/width = 1 
```
i know this isn't the same as above, but this is the correct way of annotating variables (it shows up correctly in vscode this way)